Controller/Payment/Webhook.php

```diff
@@ -237,9 +273,17 @@ protected function authorize(array $post)
                 {
                     $payment = $order->getPayment();
 
-                    if (($order->getStatus() === static::STATUS_PENDING) and
-                        (empty($payment->getLastTransId()) === true) )
+                    if ($order->getState() === static::STATE_NEW and
```

Comment:
@enishant , still formatting not correct 
if (($order->getState() === static::STATE_NEW) and
    (($order->getStatus() === static::STATUS_CANCELED) or
     ($order->getStatus() === static::STATUS_PENDING)))

Reply:
Please apply where it's not correct.